IFC4x3/Properties/r/ReferenceEnvironmentTemperature_2vgOshA4D0$ems5HSBwemr/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ReferenceEnvironmentTemperature_2vgOshA4D0ems5HSBwemr" Name="ReferenceEnvironmentTemperature" UniqueId="b9a98dab-2843-40fe-8c36-15170bea8c35" PropertyType="p_boundedvalue" PrimaryDataType="IfcRatioMeasure" />
```

Comment:
Secondary datatype should also be provided.

Reply:
It looks that all the existing bounded value properties only have Primary Data Type. The ifcDoc UI even greys the Select button for Secondary Data Type when it is a bounded value property. Shouldn't we consider the Secondary Data Type the same with Primary Data Type by default?

Reply:
an IfcPropertyBoundedValue has only one unit, and the measure type of all values (upper, lower, set point) shall be the same (enforced by where rules). So there is no secondary data type for bounded values - only IfcPropertyTableValue has two unit / measure type definitions.

NOTE: there seems to be an issue with the documentation of IfcPropertyTemplate.SecondaryMeasureType - P_BoundedValue should not be listed here, as it contradicts with the where rules at IfcPropertyBoundedValue. Should be added as a general issue on IFC4.3 (as it is not INFRA/RAIL specific).

Reply:
No need to change